server/text_generation_server/utils/paged.py

```diff
@@ -5,12 +5,37 @@
 
 from fms_extras.models.speculator import flatten_batch, apply_index_map
 
+# HF name or path to speculator model (None means no speculation will be used)
+SPECULATOR_NAME = os.getenv("SPECULATOR_NAME", None)
+
+# speculator revision
+SPECULATOR_REVISION = os.getenv("SPECULATOR_REVISION", None)
```

Comment:
What is this used for when loading?

Reply:
Looks like it's analogous to `MODEL_REVISION`, the specific commit hash of the model to load. Like one of these I think: https://huggingface.co/ibm/granite-7b-lab-accelerator/commits/main